hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetCache.java

```diff
@@ -607,11 +608,11 @@ public long getCacheCapacity() {
   }
 
   public long getNumBlocksFailedToCache() {
-    return numBlocksFailedToCache.get();
+    return numBlocksFailedToCache.longValue();
   }
 
   public long getNumBlocksFailedToUncache() {
-    return numBlocksFailedToUncache.get();
+    return numBlocksFailedToUncache.longValue();
   }
 
   public long getNumBlocksCached() {
```

Comment:
Is there any reason not to use `LongAdder` for `numBlocksCached`?

Reply:
> Is there any reason not to use `LongAdder` for `numBlocksCached`?

I'm sorry I missed this, I'll fix it right away. Thank you @tamaashu for your review.

Reply:
I fixed that.

Reply:
@tomscut  Now we can remove `import java.util.concurrent.atomic.AtomicLong` in  FsDatasetCache.java. Others look good to me.

Reply:
Thank you @tasanuma for your careful review. I will be more careful when I submit the code later. : )